src/services/item/plugins/recycled/index.ts

```diff
@@ -32,7 +34,7 @@ const plugin: FastifyPluginAsync<RecycledItemDataOptions> = async (fastify, opti
   const { db, websockets } = fastify;
   const { maxItemsInRequest = MAX_TARGETS_FOR_READ_REQUEST } = options;
 
-  const recycleBinService = new RecycledBinService();
+  const recycleBinService = new RecycledBinService(resolveDependency(ItemThumbnailService));
```

Comment:
I'm wondering if there is a specific reason of why `RecycledBinService` is not a dependency ? Could we inject the needed parameters instead of doing it here ?